src/App.css

```diff
@@ -1,38 +1,15 @@
-.App {
-  text-align: center;
+.dg_list_counter {
+  font-style: italic;
+  color: gray;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.dg_card_date-author_separator {
+  border-right: 2px solid gray;
+  padding-right: 20px;
+  margin-right: 20px;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.dg_card_date-author p {
+  text-decoration: underline;
+  color: blue;
```

Comment:
this is unneeded, this make it seem like its a link, but it should just be plain text.

Reply:
removed all that css